### PR TITLE
Add portable build to windows

### PR DESCRIPTION
**Pull request by McGiverGim**
This adds a "portable" zip build to windows (win32 and win64) releases. In this way some of the complains about having different versions installed, or about problems with the installer will be gone.

It modifies the name of the "installer" to be similar to the portable one. The linux version adds the portable word too.

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
@McGiverGim Is this for 10.7.x only? Or do we get 32bit builds for 10.8.x now too?

**Comment by McGiverGim** [comment]
> @McGiverGim Is this for 10.7.x only? Or do we get 32bit builds for 10.8.x now too?

We only build the 64 bit version. We can build the 32 bit version manually only.

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by limonspb** [comment]
@McGiverGim  thats really cool.
One question - once we release or in nightly builds, are we going to have separate downloads for portable/installer or it will be one zip folder?

**Comment by McGiverGim** [comment]
Separate. One for installer, another for portable.

**Review by limonspb — APPROVED** [review]

**Pull request merged**
